src/components/Onboarding/Screens/Apps/VotingSettings/index.js

```diff
@@ -141,37 +142,42 @@ function VotingSettings() {
     [updateField]
   )
 
-  const handleDelegatedVotingPeriodChange = useCallback(
+  const handleQuiteEndingPeriodChange = useCallback(
     value => {
       setFormError(null)
-      updateField(['voteDelegatedVotingPeriod', value])
+      updateField(['voteQuietEndingPeriod', value])
     },
     [updateField]
   )
-  const handleQuiteEndingPeriodChange = useCallback(
+
+  const handleExecutionDelayPeriodChange = useCallback(
     value => {
       setFormError(null)
-      updateField(['voteQuietEndingPeriod', value])
+      updateField(['voteExecutionDelay', value])
     },
     [updateField]
   )
 
-  const handleQuiteEndingExtensionPeriodChange = useCallback(
+  const handleDelegatedVotingPeriodChange = useCallback(
     value => {
       setFormError(null)
-      updateField(['voteQuietEndingExtension', value])
+      updateField(['voteDelegatedVotingPeriod', value])
     },
     [updateField]
   )
 
-  const handleExecutionDelayPeriodChange = useCallback(
+  const handleQuiteEndingExtensionPeriodChange = useCallback(
     value => {
       setFormError(null)
-      updateField(['voteExecutionDelay', value])
+      updateField(['voteQuietEndingExtension', value])
     },
     [updateField]
   )
 
+  const handleCloseModal = useCallback(() => setOpenSettingsModal(false), [
+    setOpenSettingsModal,
+  ])
```

Comment:
There´s no need to add the state updater function, it´s the same function between re-renders.

```suggestion
  const handleCloseModal = useCallback(() => setOpenSettingsModal(false), [])
```